src/components/Timer/index.tsx

```diff
@@ -21,18 +20,27 @@ export default function Question({
   selectedQuestionId,
 }: Props) {
   const [remainingTime, setRemainingTime] = useState(duration);
+  const [circleDasharray, setCircleDasharray] = useState('283');
+  const [alert, setAlert] = useState(false);
+  const [warning, setWarning] = useState(false);
 
   useEffect(() => {
     let timer: NodeJS.Timeout;
 
     if (running) {
       timer = setTimeout(() => {
         // allow 3s grace time to click the submit button by host
```

Comment:
Not allowing grace time anymore?

Reply:
no.. this timer doesn't support negative time value